tests/storage/ext/test_ext_sr.py

```diff
@@ -91,6 +92,34 @@ def test_coalesce(self, storage_test_vm: VM, vdi_on_ext_sr: VDI, vdi_op):
             if new_vdi is not None:
                 new_vdi.destroy()
 
+    @pytest.mark.small_vm
+    @pytest.mark.parametrize("compression", ["none", "gzip", "zstd"])
+    def test_xva_export_import(self, vm_on_ext_sr: VM, compression):
+        vm = vm_on_ext_sr
+        vm.start()
+        vm.wait_for_vm_running_and_ssh_up()
+        install_randstream(vm)
+        # 500MiB, so we have some data to check and some empty spaces in the exported image
+        vm.ssh("randstream generate -v --size 500MiB /root/data")
+        vm.ssh("randstream validate -v --expected-checksum 24e905d6 /root/data")
+        vm.shutdown(verify=True)
+        xva_path = f'/tmp/{vm.uuid}.xva'
+        imported_vm = None
+        try:
+            vm.export(xva_path, compression)
+            # check that the zero blocks are not part of the result. Most of the data is from the random stream, so
```

Comment:
```suggestion
            # check that the zero blocks are not part of the result.
            # Most of the data is from the pseudo-random stream, so
```

Reply:
To reduce the text width? Our coding style allows up to 120 chars